### DISCUSSION TITLE
Edge Function `config.regions` not working

### DISCUSSION THREAD

**Discussion by papaland-asia** [vercel/community#2059] [Help] [answered]
### Summary

I export the config with `regions: ['sin1']`, but the edge function still deployed on global.
I follow this document [https://vercel.com/docs/concepts/functions/edge-functions#edge-function-regions](https://vercel.com/docs/concepts/functions/edge-functions#edge-function-regions),
but not working, The region of the edge function still not specify to Singapore.

### My API route:
```javascript
export const config = {
  runtime: 'edge',
  regions: ['sin1'],
}
export default WebEdgeHandler().extend(videoHandler)
```
### The screenshot from dashboard
![image](https://user-images.githubusercontent.com/122956535/231783274-47e2a00b-554c-4d7e-b279-c1ced53f788c.png)
### Request headers from the Edge Function
![螢幕截圖 2023-04-13 22 05 31](https://user-images.githubusercontent.com/122956535/231926642-ec3b6277-bdf6-4920-ab4a-83066fd55d8d.png)

**Comment by amyegan** [comment]
Hi @papaland-asia. Thanks for reaching out with this info. I was not able to reproduce the problem with my test app. Please share the URL or ID of the deployment that includes this Edge Function and I will try to dig up more information about it. If you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to share, that would also be very helpful for debugging

**Reply by papaland-asia** [reply]
Hi @amyegan 
Deployment ID: F6gK3i7Udaxz6VcftxXV2X5ZFx4B
Deployment URL: https://vercel.com/papaland/papaland-web/F6gK3i7Udaxz6VcftxXV2X5ZFx4B
I can't share the website URL.

**Reply by papaland-asia** [reply, marked as answer]
Hi @amyegan 
I found that adding the Sentry integration causes the Edge Function to deploy globally
